setup.py

```diff
@@ -76,6 +76,7 @@
         'python-dateutil==2.7.3',
         'pytz==2018.4',
         'PyYAML==3.13',
+        'python-rapidjson==0.8.0',
```

Comment:
@tkaemming did we experiment with rapidjson before? Perf improvement is substantial:
rapidjson
```
calls        cumulative  file
201201    1.199           /Users/filippopacifici/snuba/snuba/datasets/__init__.py:75(<lambda>)
```

json
```
calls        cumulative  file
   201201    3.233 /Users/filippopacifici/snuba/snuba/datasets/__init__.py:48(encode)```

Reply:
We've talked about it, but I don't think we did any actual quantitative testing.